### PR TITLE
Reproduction of onnx results

### PR DESCRIPTION
**Pull request by cadurosar**
Correctly reproduced the results using onnx.

Environment
openjdk 11.0.13 2021-10-19
OpenJDK Runtime Environment JBR-11.0.13.7-1751.21-jcef (build 11.0.13+7-b1751.21)
OpenJDK 64-Bit Server VM JBR-11.0.13.7-1751.21-jcef (build 11.0.13+7-b1751.21, mixed mode)
Python 3.8.13

### CODE REVIEW

**Review by lintool — CHANGES_REQUESTED** [review]
Change to `a403a2` please? We use only six alphanumerics.

**Comment by cadurosar** [comment]
> Change to `a403a2` please? We use only six alphanumerics.

Oh did not notice, is it ok now?

File: docs/regressions-msmarco-passage-splade-pp-ed-onnx.md [hunk 1]
```diff
@@ -107,3 +107,5 @@ With the above commands, you should be able to reproduce the following results:
 ## Reproduction Log[*](reproducibility.md)
 
 To add to this reproduction log, modify [this template](../src/main/resources/docgen/templates/msmarco-passage-splade-pp-ed-onnx.template) and run `bin/build.sh` to rebuild the documentation.
+
++ Results reproduced by [@cadurosar](https://github.com/cadurosar) on 2023-05-31 (commit [`a403a2`](https://github.com/castorini/anserini/commit/https://github.com/castorini/anserini/commit/a403a2))
```
Comment (lintool) [inline docs/regressions-msmarco-passage-splade-pp-ed-onnx.md:111]:
```suggestion
+ Results reproduced by [@cadurosar](https://github.com/cadurosar) on 2023-05-31 (commit [`a403a2`](https://github.com/castorini/anserini/commit/a403a2a44af9322c7a2dbdb5240180a62398ab06))
```

File: docs/regressions-msmarco-passage-splade-pp-sd-onnx.md [hunk 2]
```diff
@@ -107,3 +107,5 @@ With the above commands, you should be able to reproduce the following results:
 ## Reproduction Log[*](reproducibility.md)
 
 To add to this reproduction log, modify [this template](../src/main/resources/docgen/templates/msmarco-passage-splade-pp-sd-onnx.template) and run `bin/build.sh` to rebuild the documentation.
+
++ Results reproduced by [@cadurosar](https://github.com/cadurosar) on 2023-05-31 (commit [`a403a2`](https://github.com/castorini/anserini/commit/https://github.com/castorini/anserini/commit/a403a2))
```
Comment (lintool) [inline docs/regressions-msmarco-passage-splade-pp-sd-onnx.md:111]:
```suggestion
+ Results reproduced by [@cadurosar](https://github.com/cadurosar) on 2023-05-31 (commit [`a403a2`](https://github.com/castorini/anserini/commit/a403a2a44af9322c7a2dbdb5240180a62398ab06))
```

File: src/main/resources/docgen/templates/msmarco-passage-splade-pp-ed-onnx.template [hunk 3]
```diff
@@ -85,3 +85,5 @@ ${effectiveness}
 ## Reproduction Log[*](reproducibility.md)
 
 To add to this reproduction log, modify [this template](${template}) and run `bin/build.sh` to rebuild the documentation.
+
++ Results reproduced by [@cadurosar](https://github.com/cadurosar) on 2023-05-31 (commit [`a403a2`](https://github.com/castorini/anserini/commit/a403a2))
```
Comment (lintool) [inline src/main/resources/docgen/templates/msmarco-passage-splade-pp-ed-onnx.template:89]:
```suggestion
+ Results reproduced by [@cadurosar](https://github.com/cadurosar) on 2023-05-31 (commit [`a403a2`](https://github.com/castorini/anserini/commit/a403a2a44af9322c7a2dbdb5240180a62398ab06))
```

File: src/main/resources/docgen/templates/msmarco-passage-splade-pp-sd-onnx.template [hunk 4]
```diff
@@ -85,3 +85,5 @@ ${effectiveness}
 ## Reproduction Log[*](reproducibility.md)
 
 To add to this reproduction log, modify [this template](${template}) and run `bin/build.sh` to rebuild the documentation.
+
++ Results reproduced by [@cadurosar](https://github.com/cadurosar) on 2023-05-31 (commit [`a403a2`](https://github.com/castorini/anserini/commit/https://github.com/castorini/anserini/commit/a403a2))
```
Comment (lintool) [inline src/main/resources/docgen/templates/msmarco-passage-splade-pp-sd-onnx.template:89]:
```suggestion
+ Results reproduced by [@cadurosar](https://github.com/cadurosar) on 2023-05-31 (commit [`a403a2`](https://github.com/castorini/anserini/commit/a403a2a44af9322c7a2dbdb5240180a62398ab06))
```

**Review by lintool — COMMENTED** [review]
Something got mangled? I fixed for you...

**Review by lintool — APPROVED** [review]

**Pull request merged**